Tue Jul  9 11:48:58 2002  Arnaud Charlet  <charlet@gnat.com>

	* gdk-window.adb: Remove reference to gdk_window_get_decorations since
	it is undefined under Windows.

	* glib-values.ads: Fix definition of GValue to conform to ms bitfields
	packing under Windows.

*** gdk-window.adb.old	Thu Apr 25 18:07:10 2002
--- gdk-window.adb	Thu Apr 25 18:07:50 2002
***************
*** 118,136 ****
        Decorations : out Gdk_Wm_Decoration;
        Success     : out Boolean)
     is
!       function Internal
!         (Window      : Gdk_Window;
!          Decorations : access Gdk_Wm_Decoration) return Gboolean;
!       pragma Import (C, Internal, "gdk_window_get_decorations");
! 
!       Tmp : aliased Gdk_Wm_Decoration;
! 
     begin
!       Success := To_Boolean (Internal (Window, Tmp'Access));
! 
!       if Success then
!          Decorations := Tmp;
!       end if;
     end Get_Decorations;
  
     ------------------------------
--- 118,128 ----
        Decorations : out Gdk_Wm_Decoration;
        Success     : out Boolean)
     is
!       pragma Warnings (Off, Window);
!       pragma Warnings (Off, Decorations);
!       pragma Warnings (Off, Window);
     begin
!       Success := False;
     end Get_Decorations;
  
     ------------------------------
*** glib-values.ads	4 Sep 2002 14:47:59 -0000	1.13
--- glib-values.ads	8 Nov 2002 11:49:02 -0000
***************
*** 136,142 ****
     --  also provided inside Gtk.Text_Iter.
  
  private
!    type GValue_Data is array (1 .. 2) of Guint64;
     type GValue is record
        g_type : GType := GType_Invalid;
        data   : GValue_Data;
--- 136,143 ----
     --  also provided inside Gtk.Text_Iter.
  
  private
! 
!    type GValue_Data is array (1 .. 20) of Guint8;
     type GValue is record
        g_type : GType := GType_Invalid;
        data   : GValue_Data;
